scrinium.csl

```diff
@@ -0,0 +1,794 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="chicago" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Scrinium: Journal of Patrology and Critical Hagiography</title>
```

Comment:
unless there are multiple journals of this name, we prefer this without subtitle, i.e. just `Scrinium`